stat_var_renaming/stat_var_renaming_constants.py

```diff
@@ -0,0 +1,358 @@
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+"""
+  This file contains various constants and helper code to generate constants
+  that are used in the Statistical Variable renaming. 
+"""
+
+import pandas as pd
+import re
+
+def capitalizeFirst(word):
+  """ Capitalizes the first letter of a string. """
+  return word[0].upper() + word[1:]
+
+def standard_name_remapper(orig_name):
+  """ General renaming function for long strings into camel case.
+
+    Text inbetween trailing parentheses is removed. 
+    Commas, dashes, and "ands" are removed. Then string is converted into camel
+    case without and spaces present.
+   """
+  # Remove any trailing parentheses.
```

Comment:
Is this safe? Can add a TODO(tjann): to check if this is safe.

Reply:
It has to be used on a case by case basis not generally